pkgs/top-level/python-packages.nix

```diff
@@ -965,8 +965,6 @@ in {
 
   backports-datetime-fromisoformat = callPackage ../development/python-modules/backports-datetime-fromisoformat { };
 
-  backports-entry-points-selectable = callPackage ../development/python-modules/backports-entry-points-selectable { };
```

Comment:
This should get an entry in python-aliases.

Reply:
forgot that now exists

Reply:
Done.